frontend/userFeaturePortal-frontend/src/app/routing/routing.module.ts

```diff
@@ -7,6 +7,9 @@ import {LoggingComponent} from "../components/logging/logging.component";
 import {HomeComponent} from "../components/home/home.component";
 import {BookComponent} from "../components/book/book.component";
 import {BibleComponent} from "../components/bibellese/bible.component";
+import {CalculatorComponent} from "../components/calculator/calculator.component";
+import {DjiStoreComponent} from "../components/dji store/dji.store.component";
+import {Dji_MiniComponent} from "../components/dji store/DJI Mini/DJI.Mini.component";
```

Comment:
Keine Leerzeichen in den Pfaden bitte. Die Ordner sollten Camel Case geschrieben werden: djiStore